package/__mocks__/react-native.js

```diff
@@ -7,20 +6,8 @@ export const Image = ReactNative.Image;
 
 Image.prefetch = () => Promise.resolve();
 
-export const FlatList = function MockedFlatList(props) {
```

Comment:
😳 Mocking the whole FlatList for all tests seems... Odd 🤔 Wonder why that was done in the first place 🤔